hoogle.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:      >= 1.18
 build-type:         Simple
 name:               hoogle
-version:            5.0.15
+version:            5.0.16
```

Comment:
As per https://github.com/ndmitchell/neil#contributions - "I only update the .cabal version just before a release" so no need to do it in a PR.